applications/ConstitutiveLawsApplication/custom_constitutive/small_strains/plastic_damage/associative_plastic_damage_model.cpp

```diff
@@ -474,11 +473,10 @@ AssociativePlasticDamageModel<TYieldSurfaceType>::CurveByPointsHardeningImplicit
 
         // Compute volumetric fracture energy in the transition by-points -> exponential
         double g_by_points = 0.5 * K0 * E0;
-        IndexType i = 1;
-        for (i; i < size_curve; ++i) {
+        for (IndexType i = 1; i < size_curve; ++i) {
             g_by_points += 0.5 * (s_by_points(i - 1) + s_by_points(i)) * (e_by_points(i) - e_by_points(i - 1));
         }
-        g_by_points = g_by_points - (0.5 * xi * s_by_points(i-1) * e_by_points(i-1) + 0.5 * (1.0 - xi) * s_by_points(i-1) * s_by_points(i-1) / C0);
+        g_by_points = g_by_points - (0.5 * xi * s_by_points(size_curve-1) * e_by_points(size_curve-1) + 0.5 * (1.0 - xi) * s_by_points(size_curve-1) * s_by_points(size_curve-1) / C0);
```

Comment:
sure about this? seems a behaviour change

Reply:
I have changed this due to a compilation error in Linux. 
The expression here is:
![image](https://github.com/KratosMultiphysics/Kratos/assets/43342945/2f721bde-6199-4353-a18e-077bc0700482)
 
 for i = n and multiplying times de fracture energy. What I then need is that these s_by_points / e_by_points components coincide with the last component in the vector. I have checked that in Kratos for the case study I had included in the PR head.
- CMD echo
![image](https://github.com/KratosMultiphysics/Kratos/assets/43342945/59bd2fdf-8fec-4446-a80e-c42256d8d4a7)
- Material json
![image](https://github.com/KratosMultiphysics/Kratos/assets/43342945/22b245df-19e2-4e9d-97d5-ea9973945959)